providers/src/airflow/providers/fab/www/package.json

```diff
@@ -0,0 +1,156 @@
+{
+  "name": "airflow-www",
+  "version": "1.0.0",
+  "description": "Apache Airflow is a platform to programmatically author, schedule and monitor workflows.",
+  "scripts": {
```

Comment:
The `build` script needs to be run to generate assets in `providers/src/airflow/providers/fab/www/static/dist`. I guess I need to modify the script `scripts/ci/pre_commit/compile_www_assets.py` to also generate assets in the fab provider? Until Airflow 3 UI will become the default UI in Airflow we'll generate twice the assets (one in core Airflow, one in Fab provider). Once the transition to the new UI done, we will only do it in Fab provider?

Reply:
Yep, sounds good.

Reply:
It should be done in a separate PR. I created a draft task in the project board: https://github.com/orgs/apache/projects/438